scripts/quests/otherAreas/An_Explorers_Footsteps.lua

```diff
@@ -0,0 +1,358 @@
+-----------------------------------
+-- An Explorer's Footsteps
+-----------------------------------
+-- Log ID: 4, Quest ID: 19
+-- This quest was changed to require a minimum amount of fame to combat RMTs POS-Hacking around to
+-- quickly earn gil. However, as this is not a legitimate concern on private servers players may
+-- complete this quest even with no fame.
```

Comment:
So, what is the actual retail implementation? We want that. 

If you want to apply popular modifications on top of this, you can also supply a module alongside it

Reply:
I just did this quest on retail with a fairly new character and didn't encounter any sort of fame requirements.  The wiki mentions neeing Selbina fame level 1 but I have done nothing but my subjob quest in Selbina and was able to complete this.

Reply:
> So, what is the actual retail implementation? We want that.
> 
> If you want to apply popular modifications on top of this, you can also supply a module alongside it

This comment was at the top of Abelard.lua (the NPC who gives the quest). I moved it from Abelard.lua to An_Explorers_Footsteps.lua but I'm not sure what it means. I've checked a few websites and can't find anything about this quest having a fame requirement added due to RMT. I think that corroborates  @gweivyth  comment about doing the quest with no fame. Can't find anything about a frame requirement for this quest anywhere so I'm not sure where the original author of that comment got their info from. 

I can remove the comment since I think it's superfluous.